internal/graph/interface.go

```diff
@@ -1,4 +1,4 @@
-//go:generate mockgen -source interface.go -destination ./mock_check_resolver_test.go -package graph CheckResolver
+//go:generate mockgen -source interface.go -destination ./mock_check_resolver.go -package graph CheckResolver
```

Comment:
Ideally, this file would be in `mocks` folder but it was a real PITA moving it, because a whole slew of things break.